harness/determined/common/experimental/experiment.py

```diff
@@ -1,4 +1,6 @@
+import base64
```

Comment:
what's the benefit of shipping to main? while it doesn't rely on anything in the feature branch, i feel it belongs in the feature branch since it's new feature work cataloged in the Q3 python SDK project. 

i feel like it makes more sense from the user's perspective as well that a bunch of SDK-related improvements would land together.

Reply:
I think we talked this through in our chat earlier enough to put out all the pros and cons. Whatever you think is best.